apps/reaper/lib/reaper/event/event_handler.ex

```diff
@@ -90,7 +90,13 @@ defmodule Reaper.Event.EventHandler do
       | technical: %{
           shapefile_dataset.technical
           | sourceFormat: hosted_file.mime_type,
-            sourceUrl: "s3://#{hosted_file.bucket}/#{hosted_file.key}"
+            extractSteps: [
```

Comment:
was unsure about replacing all extract steps here so lmk if this is a mistake

Reply:
I think this is an exceptional enough case that you shouldn't worry too much about it. We should support more of them at some point though